### PR TITLE
Topic3

### PR DESCRIPTION
**Pull request by leandrodalbo**
Hello, I am making the pull request for the key points of the topic 3. They are in the same project. 
I'll be waiting for your corrections.

### CODE REVIEW

File: topic3/src/keyPoint1/AmountConverterTester.java [hunk 1]
```diff
@@ -0,0 +1,124 @@
+package keyPoint1;
+
+import org.junit.Test;
+
+public class AmountConverterTester {
+
+	@Test
+	public void testSplitAmount() {
+		AmountConverter amountConverter = new AmountConverter("23.34");
+		System.out.println(amountConverter.getDigits() + " " + amountConverter.getCents());
```
Comment (juanip) [inline topic3/src/keyPoint1/AmountConverterTester.java]: Don't print. Use Assert.* methods to validate the expected value and the one returned by your method

Comment (juanipaccapelo-globant) [inline topic3/src/keyPoint1/AmountConverterTester.java]: Don't print. Use Assert.* methods to validate the expected value and the one returned by your method

File: topic3/src/keyPoint2/WordWrapper.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+	private String word;
+
+	public WordWrapper(int rowLength, String word) {
+		this.rowLength = rowLength;
+		this.word = word;
+	}
+
+	public void print() {
+
+		switch (rowLength) {
```
Comment (juanipaccapelo-globant) [inline topic3/src/keyPoint2/WordWrapper.java]: 60, 7, 3 and 5 are just examples. Consider that rowLength can be any integer between 1 and 100. Also word can be any string

Comment (leandrodalbo) [reply topic3/src/keyPoint2/WordWrapper.java]: Hey, I solved this using the strategy design pattern.

File: topic3/src/keyPoint1/AmountConverterTester.java (hunk 1 above)
Comment (leandrodalbo) [reply topic3/src/keyPoint1/AmountConverterTester.java]: Hello, I have corrected it all, now it is with Assert methods in all the key points.

**Review by juanip — APPROVED** [review]
Okay, you can merge

**Pull request merged**